pages/static/serviceworker.js

```diff
@@ -9,3 +10,67 @@ AMP_SW.init({
     assets: [],
   },
 });
+
+const CACHE_NAME = 'AMP-DEV-SEARCH-CACHE';
+const LATEST_QUERY_URL = '/search/latest-query';
+
+
+// Match search request url
+const matchDo = ({url}) => {
```

Comment:
Valid for all functions here: use real named functions, so this would become:

```js
function matchDo(url) {
}
```

Also why doing so try to find proper function names as `Do` and `Cq` are not really obvious for someone reading the code for the first time. 🙂 Maybe scope all names to something starting with `search`.

Reply:
Thanks for the feedback! Everything clean now :-)